modules/samples/data-populator/Populator/src/org/wso2/carbon/registry/samples/populator/CategorizeArtifacts.java

```diff
@@ -11,29 +11,39 @@
 import org.wso2.carbon.registry.ws.client.registry.WSRegistryServiceClient;
 import org.wso2.carbon.registry.core.Registry;
 import org.wso2.carbon.registry.core.session.UserRegistry;
+import org.wso2.carbon.registry.samples.populator.utils.LifeCycleManagementClient;
+import org.wso2.carbon.registry.samples.populator.utils.UserManagementClient;
+import org.wso2.carbon.user.mgt.stub.types.carbon.ClaimValue;
 
 import javax.xml.namespace.QName;
 import java.io.File;
+import java.lang.System;
 import java.util.ArrayList;
 import java.io.FileNotFoundException;
 import java.io.IOException;
 import java.io.BufferedReader;
 import java.io.FileReader;
 
-
+/*
+* This class add categories,tags to rest and soap services, and create publisher and
+* store user roles.
+ */
 public class CategorizeArtifacts {
+    private static String cookie;
     private static ConfigurationContext configContext = null;
-
+    private static LifeCycleManagementClient lifeCycleManagementClient;
     private static final String CARBON_HOME = System.getProperty("carbon.home");
     private static final String axis2Repo = CARBON_HOME + File.separator + "repository" + File.separator + "deployment" + File.separator + "client";
```

Comment:
Refer the commit
https://github.com/Rajith90/product-greg/commit/0d2c0d6bbe023fa08a8942fd6dd7a4f2332dc117